app/models/setting/auth.rb

```diff
@@ -4,7 +4,7 @@ def self.default_settings
       set('oauth_active', N_("Foreman will use OAuth for API authorization"), false, N_('OAuth active')),
       set('oauth_consumer_key', N_("OAuth consumer key"), '', N_('OAuth consumer key'), nil, {:encrypted => true}),
       set('oauth_consumer_secret', N_("OAuth consumer secret"), '', N_("OAuth consumer secret"), nil, {:encrypted => true}),
-      set('oauth_map_users', N_("Foreman will map users by username in request-header. If this is set to false, OAuth requests will have admin rights."), true, N_('OAuth map users')),
+      set('oauth_map_users', N_("Foreman will map users by username in request-header. If this is set to No, OAuth requests will have admin rights."), true, N_('OAuth map users')),
```

Comment:
I think this is a boolean but the UI maps that to yes/no I think. Note the default is `true`, not `yes`.

Reply:
That's right. It's still boolean at the DB side. From UI prespective, the default value Yes because it's mapped to true. 
I don't think to have three possible (true, yes and no) values for boolean from user's perspective and I that's is what I tried to solve with this PR.

Reply:
It would be good to check what API uses, we should be consistent with this since the help is shared for both. I'd lean to use Yes/No since that's something that user see in the UI. My assumption is even if the default is true, it's displayed as Yes

Reply:
I've checked with API and even with the Hammer and situation is little tricky. Both hammer and API shows true/false values however, both of them accept Yes/No values too... So from my point of view I would like to avoid to switch to true/false logic at the API due to usage by external services/programs. 
So I think that best solution will be improve the in the way, where that we're accept both values. Something like: 
`If this is set to No (false)...`

What do you think @ares and @ekohl ?

Reply:
You could avoid the literal values and say something "when enabled" or "when disabled".

Reply:
Good idea! I've updated the PR according to that.